### ISSUE TITLE
Latest 26.0.0 (scheduled) build is broken

### ISSUE THREAD

**Issue by waja** [hoellen/docker-nextcloud#38]
From the logs:
```
Error loading shared library libssl.so.3: No such file or directory (needed by /usr/sbin/nginx)
Error loading shared library libcrypto.so.3: No such file or directory (needed by /usr/sbin/nginx)
```

So nginx is not starting ...

**Comment by waja** [comment]
For all affected users: https://github.com/hoellen/docker-nextcloud/pkgs/container/nextcloud/79188091?tag=sha256-12f0914f59e8725be9003726f7edc42005f5eaaedbdaa4be1ddac2727f6fddc8.sig seems safe to use

**Comment by mrplow** [comment]
Thanks @waja 
setting my compose file to use
`    image: ghcr.io/hoellen/nextcloud@sha256:12f0914f59e8725be9003726f7edc42005f5eaaedbdaa4be1ddac2727f6fddc8`
fixed this for me while waiting for this to be resolved

**Comment by hoellen** [comment]
The official nginx images have been rebuilt with Alpine 3.17 instead of 3.16. 
Since Alpine 3.17 is now shipped with OpenSSL 3.0, the mentioned libraries are not shared between the images. So we need to update the Alpine version as well. Images should now be up-to-date and the issue fixed.
